visual.css

```diff
@@ -0,0 +1,128 @@
+body {
+		display: flex;
+		flex-direction: column;
+		min-height: 100vh;
+		text-align: center;
+		background-image: linear-gradient(260deg, #9E9E9E 0%, #58266133 100%);
+		box-sizing: border-box;
+		margin: 0;
+}
+.header {
+		display: flex;
+		width: 100%;
+		height: 200px;
+		background:url(https://www.nasa.gov/sites/default/files/thumbnails/image/stsci-h-p2001a-m-2000x1500_0.png) center/cover;
+		font-style: italic;
+		color: white;
+		border:2px solid goldenrod;		
+}
+.bar {
+		flex-grow:1;
+		color: white;
+		text-align:left;
+		max-height: 200px;
+		border:2px solid goldenrod;
+}
+.bar a {
+		margin:12px;
+		padding:16px;
+		word-spacing:5px;
+		font-size: 1em;
+		color: white;
+		text-decoration:none;
+		border:2px solid goldenrod;
+		line-height:2.5;
+}
+.bar p {
+		text-align: center;
+		font-size: 4em;	
+		font-style:italic;
+		letter-spacing:5px;
+		word-spacing:15px;
+		margin-top: 25px;
+		border:2px solid goldenrod;
+}
+.whole-universe {
+		flex: 1 0 auto;
+		display: flex;
+		border:2px solid goldenrod;
+}    
+.voating {
+		width: 150px;
+		list-style: none;
+		text-align: left;
+		order: -1;
+		background: yellow;
+		margin-top: 5px;
+		padding:20px;
+		max-height:29em;
+		border:2px solid goldenrod;	
+}
+.voating li {
+		line-height: 2.5;
+}
+.voating h3 {
+		flex:1 1 100%;
+		text-align:center;
+		margin:3px;
+		padding:13px;
+		border:2px solid goldenrod;
+}
+.content {
+		flex: 1 0 70%;	
+		border:2px solid goldenrod;
+		text-align:left;
+		margin:10px;
+		line-height:1.5;
+		font-style: oblique;
+		font-size:20px;	
+}
+.content img {
+		max-width:80%;
+		max-height:25%;
+		border-bottom:3px solid rgba(10,8,9,0.85); 
```

Comment:
this border looks strange: remove it at all or do it across all article's width

Reply:
> this border looks strange: remove it at all or do it across all article's width

changing for normal appearance